### ISSUE TITLE
libcanberra: broken

### ISSUE THREAD

**Issue by pulux** [voidlinux/void-packages#1126]
```
./xbps-src -N pkg libcanberra
=> libcanberra-0.30_6: the following build options are set:
   systemd: Enable support for systemd (OFF)
=> libcanberra-0.30_6: building ...
   [host] libtool>=0: found libtool-2.4.2_10 in /host/binpkgs.
   [host] pkg-config>=0: found pkg-config-0.28_3 in /host/binpkgs.
   [host] intltool>=0: found intltool-0.51.0_1 in /host/binpkgs.
   [target] gtk+-devel>=0: found gtk+-devel-2.24.27_2 in /host/binpkgs.
   [target] gtk+3-devel>=0: found gtk+3-devel-3.14.9_2 in /host/binpkgs.
   [target] libvorbis-devel>=0: found libvorbis-devel-1.3.5_1 in /host/binpkgs.
   [target] libltdl-devel>=0: found libltdl-devel-2.4.2_10 in /host/binpkgs.
   [target] gstreamer1-devel>=0: found gstreamer1-devel-1:1.4.5_1 in /host/binpkgs.
   [target] alsa-lib-devel>=0: found alsa-lib-devel-1.0.29_1 in /host/binpkgs.
   [target] tdb-devel>=0: found tdb-devel-1.3.4_1 in /host/binpkgs.
   [target] pulseaudio-devel>=0: found pulseaudio-devel-6.0_3 in /host/binpkgs.
   [target] eudev-libudev-devel>=183: not found.
   [target] vala-devel>=0: found vala-devel-0.26.2_1 in /host/binpkgs.
=> ERROR: libcanberra-0.30_6: required target dependency 'eudev-libudev-devel>=183' cannot be resolved!
```
in repo is `eudev-libudev-devel-2.1.1_6` not >= 183
```
xbps-query -Rs eudev-libudev
[*] eudev-libudev-2.1.1_6       Gentoo's fork of systemd-udev (enhanced userland device daemon) - runtime library
[-] eudev-libudev-devel-2.1.1_6 Gentoo's fork of systemd-udev (enhanced userland device daemon) - development file
```

**Comment by bougyman** [comment]
Updating now.

**Comment by bougyman** [comment]
Should be fixed by #1129 and #1130
